docs/sphinx/_toc.yml.in

```diff
@@ -4,14 +4,17 @@ root: index
 subtrees:
   - caption: Install
     entries:
-    - file: installation    
+    - file: installation
   - caption: API reference
     entries:
     - file: datatypes
     - file: api-reference
     - file: ext-reference
     - file: ext-ops
     - file: logging-heuristics
+  - caption: Utilities
```

Comment:
I recommend calling this caption How To so it aligns with the name of the folder

Reply:
Changed